Feature/add message binding to channel documentation

sam0r040 opened the pull request:
This PR extends the springwolf-ui to display message binding information which is added in springwolf-core in https://github.com/springwolf/springwolf-core/pull/130.
It also moves the editable header example to the example view so that everything that can be edited and send to the backend to produce messages is now on one screen. 

Changes:
- A new tab "Message Bindings" is added to display the message binding spec (similar to operation binding)
- The textbox for editing the headers example has been moved to the examples tab
- A ready-only view of the headers example has been added to the headers tab
- A textbox for editing the message binding example has been added to the example tab
- The content of the message binding example textbox will be send to the backend when producing messages

sam0r040 committed:
feat: Add new tab with message binding to channel documentation and add header and message binding example to example tab

sam0r040 committed:
Add message binding to the request made by PublisherService so that the backend can use the message binding to construct a message

sam0r040 committed:
Improve mapping of message bindings to be more resilient if a backend creates a specification without message binding

timonback commented:
Great contribution!

The data in the `message binding` tab seems off and includes springwolf-ui internal data, is it intended that way?
https://deploy-preview-35--mellow-manatee-3041fb.netlify.app/springwolf-ui/asyncapi-ui.html#channel-kafka-example-producer-topic-subscribe-AnotherPayloadDto

In terms of layout, I wonder if the items in example should be sorted the other way round. As an user, I am mostly interested in the payload/data, which I tend to expect first. What do you think?

timonback commented:
@stavshamir I'd like to motivate once more a common code formatting style. To me, it does not matter, which formatting is used, but it should be consistent. It also avoids the changed lines due to added/missing spaces.
No need to change the content of the PR, but I attempt to get to a common formatting style. One example is present in #14

sam0r040 committed:
Rename Binding in example tab to Message Binding and add full payload to log statement when producing messages

timonback committed:
Display raw message bindings

Regression: Previously included mapped ui attributes
chore: Improve [protocol: string]: any typings

Co-authored-by: david.mueller@codecentric.de
